test/function.localstack.test.ts

```diff
@@ -152,14 +210,7 @@ runtimeTestSuite("functionStack", (testResource, _stack, _app) => {
     (parent) => {
       const create = () => {
         const val = "a";
-        return new Function(
-          parent,
-          "function",
-          {
-            timeout: Duration.seconds(20),
-          },
-          async () => val
-        );
+        return standardFunction(parent, async () => val);
```

Comment:
Could be confused with standard step functions lol.

Reply:
Much cleaner though

Reply:
Maybe `functionWithSharedRole`

Reply:
Ha, sure.